department-of-veterans-affairs/caseflow
# Remove Legacy VACOLS Hearing Day Logic

Resolves #8991 

### Description
Removed all logic that fetched hearing days from VACOLS. All hearing days are now stored in Caseflow.

### Acceptance Criteria

### Testing Plan
1. Go to View Hearings, Daily Docket and Assign Hearings
2. Confirm all hearing days and child hearings are properly fetched and displayed.
